src/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensSchema.cs

```diff
@@ -117,14 +107,17 @@ public SemanticTokensSchema(IReadOnlyDictionary<string, string> tokenTypeMap)
 
         public static SemanticTokensSchema GetSchema(bool clientSupportsVisualStudioExtensions)
             => clientSupportsVisualStudioExtensions
-                ? s_vsTokenSchema
+                ? LegacyTokensSchemaForLSIF
                 : s_pureLspTokenSchema;
 
         public static SemanticTokensSchema LegacyTokenSchemaForRazor
-            => s_vsTokenSchema;
+            => LegacyTokensSchemaForLSIF;
 
-        public static SemanticTokensSchema LegacyTokensSchemaForLSIF
-            => s_vsTokenSchema;
+        public static SemanticTokensSchema LegacyTokensSchemaForLSIF { get; } = new(ClassificationTypeNames.AllTypeNames
```

Comment:
```suggestion
        /// <summary>
        /// A schema for mapping classification type names to VS LSP token names.  This maps a few classification type names
        /// directly to LSP semantic token types, but otherwise generally returns the classification type name as a custom token type.
        /// </summary>
        public static SemanticTokensSchema LegacyTokensSchemaForLSIF { get; } = new(ClassificationTypeNames.AllTypeNames
```